src/strategies/synthetix/index.ts

```diff
@@ -0,0 +1,56 @@
+import { formatUnits } from "@ethersproject/units";
+import { multicall } from "../../utils";
+
+export const author = "andytcf";
+export const version = "0.0.1";
+
+const synthetixStateAbi = [
+  {
+    constant: true,
+    inputs: [{ name: "", type: "address" }],
+    name: "issuanceData",
+    outputs: [
+      { name: "initialDebtOwnership", type: "uint256" },
+      { name: "debtEntryIndex", type: "uint256" },
+    ],
+    payable: false,
+    stateMutability: "view",
+    type: "function",
+  },
+];
+
+const synthetixStateContractAddress =
+  "0x4b9Ca5607f1fF8019c1C6A3c2f0CC8de622D5B82";
```

Comment:
Is there others same smart contract with different address, for example testnet contracts? If yes we may want to have this value as part of the strategy options.

Reply:
sure, i've seen the implementation elsewhere but I think for now we want to only include the main-net contract